data/presets/disc_golf/hole.json

```diff
@@ -0,0 +1,15 @@
+{
+    "icon": "temaki-disc_golf_basket",
+    "fields": [
+        "name",
+        "ref_golf_hole",
```

Comment:
:thinking: while this seems to work fine in English (where the term _hole_ is also used in the context of disc golf courses), it doesn't work in other Languages. For example, in German, different terms are used for disc golf and regular golf (_Bahn_ vs. _Loch_).

I think it would be better to add a new field for `ref_disc_golf_hole` or alternatively to use the generic `ref` field.

Reply:
Thanks for letting me know about this, I've added a new `ref_disc_golf_hole` like you suggested.